plugin/trino-pinot/pom.xml

```diff
@@ -34,6 +34,11 @@
                 <artifactId>jersey-common</artifactId>
                 <version>2.30</version>
             </dependency>
+            <dependency>
+                <groupId>org.osgi</groupId>
+                <artifactId>org.osgi.core</artifactId>
+                <version>6.0.0</version>
```

Comment:
Why do we need this?

Reply:
There is a upper bound violation being detected on newer dependency plugin.